llumnix/backends/vllm/executor.py

```diff
@@ -149,6 +151,11 @@ def _init_workers_ray(self, placement_group: "PlacementGroup",
         self._run_workers("load_model",
                           max_concurrent_workers=self.parallel_config.
                           max_parallel_loading_workers)
+        self._run_workers("reserver_memory_for_migration_backend",
```

Comment:
reserver -> reserve, also i think maybe reserver_memory_for_migration is enough? Because migration is our feature.

Reply:
yes, accept